src/main/kotlin/cache/loaders/OverlayLoader.kt

```diff
@@ -59,6 +60,14 @@ class OverlayLoader(cacheLibrary: CacheLibrary) {
             } else if (opcode == 7) {
                 val secondaryColor: Int = inputStream.read24BitInt()
                 def.secondaryRgbColor = secondaryColor
+            } else if (opcode == 8) {
+                // Handle opcode 8 introduced in newer cache revisions
```

Comment:
this comment will age very quickly & describes what can be seen in the line above, so probably best to leave it out